frontend/src/components/pages/TourDeFoos/TourDeFoos.tsx

```diff
@@ -13,49 +14,51 @@ import {HomeMenu} from "../HomeMenu/HomeMenu";
 import {TournamentRegistrations} from "../TournamentRegistrations";
 
 const theme = createMuiTheme({
-    palette: {
-        primary: {
-            main: grey[800],
-            contrastText: "#E3E152",
-        },
+  palette: {
+    primary: {
+      main: grey[800],
+      contrastText: "#E3E152",
     },
+  },
 });
```

Comment:
Coś z formatowaniem nie tak :) pamiętaj o sformatowaniu przed commitem

Reply:
dziwne, bo wszystko miałem zrobione npm run formatem

Reply:
to właśnie może te poprawki są po moim dobrym formatowaniu :-)